src/ContentMessages.ts

```diff
@@ -351,7 +352,12 @@ export default class ContentMessages {
         text: string,
         matrixClient: MatrixClient,
     ): Promise<ISendEventResponse> {
-        const prom = matrixClient.sendStickerMessage(roomId, threadId, url, info, text).catch((e) => {
+        const prom = doMaybeLocalRoomAction(
+            roomId,
+            (actualRoomId: string) => matrixClient.sendStickerMessage(actualRoomId, threadId, url, info, text),
+            matrixClient,
+        );
+        prom.catch((e) => {
```

Comment:
Does
```
prom.catch((e) => { ... });
return prom;
```
have the same semantics as 
```
return prom.catch((e) => { ... });
```
?

Reply:
No, in the former the caller can catch too